public/app/api/clients/provisioning/endpoints.gen.ts

```diff
@@ -1096,17 +1096,23 @@ export type ResourceList = {
   kind?: string;
   metadata?: ListMeta;
 };
+export type FieldError = {
+  detail?: string;
+  type: string;
+};
 export type TestResults = {
   /** APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources */
   apiVersion?: string;
   /** HTTP status code */
   code: number;
-  /** Optional details */
-  details?: Unstructured;
-  /** Error descriptions */
-  errors?: string[];
+  /** Field related errors */
+  fields?: {
```

Comment:
@Clarity-89 -- we could also structure as a list where field (path?) is an optional value.  That might make more sense anyway

Reply:
I think this is fine (for now at least) the returned fields do not always 100% match our form fields, so we'll still need to do some processing on FE. Could omit the `spec` part from the path, but if not possible, also fine.